gtsfm/frontend/verifier/colmap_two_view_geometry_verifier.py

```diff
@@ -0,0 +1,195 @@
+            Both SIMPLE_PINHOLE and SIMPLE_RADIAL use 1 focal length.
+            """
+            focal_length = camera_intrinsics.fx()
+            cx, cy = camera_intrinsics.px(), camera_intrinsics.py()
+
+            # TODO (johnwlambert): use more accurate proxy?
+            width = int(cx * 2)
+            height = int(cy * 2)
+            logger.info("Estimated height and width: %d,%d", height, width)
+
+            camera_dict = pycolmap.Camera(
+                model="SIMPLE_PINHOLE",
+                width=width,
+                height=height,
+                params=[focal_length, cx, cy],
+            )
+            return camera_dict
+
+        camera_dict1 = get_pycolmap_camera_dict(camera_intrinsics_i1)
+        camera_dict2 = get_pycolmap_camera_dict(camera_intrinsics_i2)
+
+        result_dict = pycolmap.two_view_geometry_estimation(
+            points2D1=uv_i1,
+            points2D2=uv_i2,
+            camera1=camera_dict1,
+            camera2=camera_dict2,
+            max_error_px=self._estimation_threshold_px,
+            min_inlier_ratio=MIN_INLIER_RATIO,
+            min_num_trials=MIN_NUM_TRIALS,
+            max_num_trials=MAX_NUM_TRIALS,
+            confidence=CONFIDENCE,
+        )
+        return result_dict
+
+    def verify(
+        self,
+        keypoints_i1: Keypoints,
+        keypoints_i2: Keypoints,
+        match_indices: np.ndarray,
+        camera_intrinsics_i1: Cal3Bundler,
+        camera_intrinsics_i2: Cal3Bundler,
+    ) -> Tuple[Optional[Rot3], Optional[Unit3], np.ndarray, float]:
+        """Performs verification of correspondences between two images to recover the relative pose and indices of
+        verified correspondences.
+
+        Args:
+            keypoints_i1: detected features in image #i1.
+            keypoints_i2: detected features in image #i2.
+            match_indices: matches as indices of features from both images, of shape (N3, 2), where N3 <= min(N1, N2).
+            camera_intrinsics_i1: intrinsics for image #i1.
+            camera_intrinsics_i2: intrinsics for image #i2.
+
+        Returns:
+            Estimated rotation i2Ri1, or None if it cannot be estimated.
+            Estimated unit translation i2Ui1, or None if it cannot be estimated.
+            Indices of verified correspondences, of shape (N, 2) with N <= N3. These are subset of match_indices.
+            Inlier ratio of w.r.t. the estimated model, i.e. the #final RANSAC inliers/ #putatives.
+        """
+        if match_indices.shape[0] < self._min_matches:
+            logger.info("[GRIC] Not enough correspondences for verification.")
+            return self._failure_result
+
+        uv_i1 = keypoints_i1.coordinates[match_indices[:, 0]]
+        uv_i2 = keypoints_i2.coordinates[match_indices[:, 1]]
+
+        result_dict = self.__estimate_two_view_geometry(uv_i1, uv_i2, camera_intrinsics_i1, camera_intrinsics_i2)
+
+        success = result_dict["success"]
+        if not success:
```

Comment:
I dont think writing `if not result_dict["success"]` is losing any readability.

Reply:
Good point, I'll update